src/api-service/__app__/onefuzzlib/notifications/main.py

```diff
@@ -97,6 +97,32 @@ def get_notifications(container: Container) -> List[Notification]:
     return Notification.search(query={"container": [container]})
 
 
+def get_regression_report_task(report: RegressionReport) -> Optional[Task]:
+    if report.crash_test_result:
+        if report.crash_test_result.crash_report:
+            return Task.get(
+                report.crash_test_result.crash_report.job_id,
+                report.crash_test_result.crash_report.task_id,
+            )
+        if report.crash_test_result.no_repro:
+            return Task.get(
+                report.crash_test_result.no_repro.job_id,
+                report.crash_test_result.no_repro.task_id,
+            )
+    elif report.original_crash_test_result:
```

Comment:
Since `crash_test_result` is a required field, but `original_` is `Optional`, won't this be unreachable?

Reply:
Good catch.